frontend/src/App.svelte

```diff
@@ -90,6 +93,18 @@
     document.body.classList[darkmode ? "add" : "remove"]("dark");
   }
 
+  function search(searchText) {
+    const restaurants = document.querySelectorAll('.restaurant');
```

Comment:
Not sure if this is best svelte approach - we are modifying data out-of-sync with svelte.
For example it breaks switching locations.

Theres already filtering by location which we can extend: https://github.com/trnila/assistant/blob/53d27f6d1c0c8e6d55fa011f409e5621bc689602/frontend/src/App.svelte#L142C1-L143C1